common/readargs.go

```diff
@@ -45,6 +47,30 @@ func (r *ReadArgs) Encode(trustDomains []*TrustDomainConfig) (out EncodedReadArg
 	return
 }
 
+// Bucket returns the bucket index that a read requests, or -1 for invalid args.
+func (r *ReadArgs) Bucket() int {
```

Comment:
Added the function for debugging. it was nice to be able to recover the bucket i was expecting a write to go into / a read to be of as it left the RequestManager.  I'll add a test.